components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -2080,58 +2067,9 @@ void RewardsServiceImpl::OnNotificationTimerFired() {
     return;
   }
 
-  GetReconcileStamp(base::BindOnce(
-      &RewardsServiceImpl::MaybeShowAddFundsNotification, AsWeakPtr()));
   FetchPromotions(base::DoNothing());
 }
 
-void RewardsServiceImpl::MaybeShowNotificationAddFunds() {
-  if (!Connected()) {
-    return;
-  }
-
-  bat_ledger_->HasSufficientBalanceToReconcile(
```

Comment:
I can't seem to find any other references to `HasSufficientBalanceToReconcile()`? Maybe we can remove that, too. And also `Contribution::HasSufficientBalance()`, `ContributionMonthly::HasSufficientBalance()`, etc. 🙂

Reply:
Yep, good point. Removed.